app/jobs/set_content_type_job.rb

```diff
@@ -50,30 +51,29 @@ def cocina_update_attributes(cocina_object)
     end
   end
 
-  # If the new content type is a book, we need to set the viewing direction attribute in the cocina model
+  # If the new content type is a book or image, we need to set the viewing direction attribute in the cocina model
   def member_orders
-    return [] unless @new_content_type.start_with?('book')
-
-    viewing_direction = if @new_content_type == 'book (ltr)'
-                          'left-to-right'
-                        else
-                          'right-to-left'
-                        end
-    [{ viewingDirection: viewing_direction }]
+    return [] unless may_have_direction? && @viewing_direction.present?
+
+    [{ viewingDirection: @viewing_direction }]
+  end
+
+  def may_have_direction?
```

Comment:
viewing direction set from the new drop down, and not looked up as before based on a content type value